### ISSUE TITLE
Add PlatformIO support?

### ISSUE THREAD

**Issue by skorokithakis** [AndunHH/spacemouse#14]
Would you be willing to add PlatformIO support? It's basically as easy as adding a `platformio.ini` file with contents like:

```
[env:micro]
platform = atmelavr
board = micro
framework = arduino
```

and moving the source files into a `src/` directory.

**Comment by AndunHH** [comment]
Is it really that simple? We changed the board.txt file in the arduino IDE so the board immitates the 3dconnexion spacemouse.

How is this done in platform.io? 

I'd appreciate a merge request, if you can show me in a fork how to do that.

**Comment by skorokithakis** [comment]
Yep, I'll look into a PR, thank you!

**Comment by AndunHH** [comment]
Issue fixed with #23
